types/node/index.d.ts

```diff
@@ -2126,20 +2126,20 @@ declare module "url" {
     }
 
     export class URLSearchParams implements Iterable<string[]> {
```

Comment:
Sorry to comment again, but I think we should use also `Iterable<[string, string]>` here...

Reply:
Thanks, added.